entrypoint.js

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+"use strict";
+
+// This is the extension entrypoint module, which imports extension.js, the actual extension code.
+//
+// This is in a separate file so we can properly measure extension.js load time.
+
+let perfStats = {
+    loadStartTime: Date.now(),
+    loadEndTime: undefined
+};
+
+Object.defineProperty(exports, "__esModule", { value: true });
+
+const extension = require("./dist/extension");
+
+async function activate(ctx) {
+    return await extension.activateInternal(ctx, perfStats);
+}
+
+exports.activate = activate;
```

Comment:
I don't see this being imported anywhere. Are these used by AzureTools?

Reply:
Yes.  Added comment.

Reply:
Ah, sounds good.